src/resources/SearchUsageMetrics/SearchHubs/SearchHubsInterface.ts

```diff
@@ -47,6 +47,13 @@ export interface ListSearchHubsParams extends Paginated {
      * @default `0`
      */
     page?: number;
+
+    /**
+     * The minimum queries a search hub must have to be listed
+     * 
+     * @default '1'
+     */
+    minimumQueries?: number;
```

Comment:
Is it a string or a number? I'm confused 🤔 
![image](https://github.com/coveo/platform-client/assets/35579930/932ae6b0-fa34-4d4c-b847-2507a9cc8b9e)

Reply:
Likely a string as the doc indicates, fixed this